scripts/tests/test_suites.sh

```diff
@@ -107,16 +110,14 @@ for j in "${iter_array[@]}"; do
         # tee expeditiously; otherwise it will buffer things up and we
         # will never see the string we want.
 
-        # Clear out our temp files so we don't accidentally do a stale
-        # read from them before we write to them.
-        rm -rf /tmp/"$application"-log
-        touch /tmp/"$application"-log
+        application_log_file=/tmp/test_suites_app_logs/"$application-$i-$j"-log
+        touch "$application_log_file"
         rm -rf /tmp/pid
         (
             stdbuf -o0 "${test_case_wrapper[@]}" out/debug/standalone/chip-"$application"-app &
             echo $! >&3
-        ) 3>/tmp/pid | tee /tmp/"$application"-log &
-        while ! grep -q "Server Listening" /tmp/"$application"-log; do
+        ) 3>/tmp/pid | tee "$application_log_file" &
+        while ! grep -q "Server Listening" "$application_log_file"; do
             :
         done
         # Now read $background_pid from /tmp/pid; presumably it's
```

Comment:
Is it worth tee-ing the output of the `chip-tool tests` command below into `/tmp/test_suites_app_logs/chip-tool-"$i-$j"-log` or something like that?  Otherwise we get a clean version of the server log, but not the client log.

Reply:
Done. Also added log file for pairing step.